ci/cloudbuild/dockerfiles/fedora-latest-cmake.Dockerfile

```diff
@@ -198,7 +198,7 @@ RUN curl -fsSL https://github.com/googleapis/google-cloud-cpp/archive/v2.33.0.ta
         -DCMAKE_CXX_STANDARD=17 \
         -DGOOGLE_CLOUD_CPP_ENABLE_CLANG_ABI_COMPAT_17=ON \
         -DBUILD_SHARED_LIBS=yes \
-        -DGOOGLE_CLOUD_CPP_ENABLE="bigquerycontrol,bigquery" \
+        -DGOOGLE_CLOUD_CPP_ENABLE="bigquerycontrol,bigquery,opentelemetry" \
```

Comment:
To use the mock libraries...

I imagine we want `-DGOOGLE_CLOUD_CPP_WITH_MOCKS=ON`.

(Also I cannot remember if that builds the `mocks` library or not. I know we can explicitly add `mocks` to the enable string. So we might want to do that.)

Reply:
Oh, good question. The mocks are not available in vcpkg.

I do not see `mocks` as a feature, and library mocks (eg for BQ) are turned off: 

https://github.com/microsoft/vcpkg/blob/master/ports%2Fgoogle-cloud-cpp%2Fportfile.cmake#L49

Maybe this is doomed and you should ignore me.

Reply:
Yeah, I agree.